metatensor-torch/src/atomistic/model.cpp

```diff
@@ -839,6 +839,19 @@ void metatensor_torch::load_model_extensions(
     }
 }
 
+std::string metatensor_torch::read_model_metadata(std::string path) {
+    auto reader = caffe2::serialize::PyTorchStreamReader(path);
+    if (!reader.hasRecord("extra/model-metadata")) {
+        C10_THROW_ERROR(ValueError,
+            "could not find model metadata in file at '" + path + "'"
```

Comment:
```suggestion
            "could not find model metadata in file at '" + path + "', did you export your model with metatensor-torch >=0.6?"
```

or we could check the export version of metatensor-torch to give a better error message in this case